### PR TITLE
Changed to reflect default cookbook generate behavior to policyfile

### PR DESCRIPTION
**Pull request by jasonnalewak**
### Description

Correcting the cookbook generate doc to align with the default behavior of generating a policyfile instead of a berksfile

### Issues Resolved

None

### CODE REVIEW

**Comment by IanMadd** [comment]
Hi @jasonnalewak, can you add your DCO signoff 
Thanks!
https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco

File: content/ctl_chef.md [hunk 1]
```diff
@@ -396,8 +396,9 @@ This subcommand has the following options:
 
 `-b`, `--berks`
 
-:   Create a Berksfile in the cookbook. Default value: enabled. This is
-    disabled if the `--policy` option is given.
+:   Create a Berksfile in the cookbook. Default value: disabled. This value
```
Comment (kagarmoe) [inline content/ctl_chef.md]:
```suggestion
:   Create a Berksfile in the cookbook. Default: disabled. Optional.
```

File: content/ctl_chef.md [hunk 2]
```diff
@@ -396,8 +396,9 @@ This subcommand has the following options:
...
+    must be specified to override the default creation of a policyfile
```
Comment (kagarmoe) [inline content/ctl_chef.md]:
```suggestion
    Use to create a berksfile by overriding default creation of a Policyfile.
```

File: content/ctl_chef.md [hunk 3]
```diff
@@ -396,8 +396,9 @@ This subcommand has the following options:
...
+    instead of a berksfile.
```
Comment (kagarmoe) [inline content/ctl_chef.md]:
```suggestion
```

File: content/ctl_chef.md (hunk 2 above)
Comment (kagarmoe) [reply content/ctl_chef.md]: Apologies if I messed with a required line length.

File: content/ctl_chef.md [hunk 4]
```diff
@@ -430,7 +431,7 @@ This subcommand has the following options:
 `-P`, `--policy`
 
 :   Create a Policyfile in the cookbook instead of a Berksfile. Default
```
Comment (kagarmoe) [inline content/ctl_chef.md]:
```suggestion
:   Create a Policyfile in the cookbook instead of a Berksfile.
```

File: content/ctl_chef.md [hunk 5]
```diff
@@ -430,7 +431,7 @@ This subcommand has the following options:
...
-    value: disabled.
+    value: enabled.
```
Comment (kagarmoe) [inline content/ctl_chef.md]:
```suggestion
    Default: enabled.
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]

**Comment by jasonnalewak** [comment]
Sorry I'm not used to doing the DCO for commits.  I rebased the commits for the suggested changes to add the DCO sign offs.  If I broke the pipeline I'm sorry :(

**Review by kagarmoe — APPROVED** [review]
tenor-249198068

**Pull request merged**